app/controllers/hiring_staff/vacancies/job_specification_controller.rb

```diff
@@ -55,12 +55,13 @@ def set_up_job_specification_form
   def job_specification_form_params
     persist_nqt_job_role_to_nqt_attribute(:job_specification_form)
     strip_empty_checkboxes(:job_specification_form, [:working_patterns, :job_roles])
+    convert_subject_ids_to_subjects_array
     params.require(:job_specification_form)
           .permit(:job_title,
                   :subject_id, :first_supporting_subject_id, :second_supporting_subject_id,
```

Comment:
Does `convert_subject_ids_to_subjects_array` make this line redundant?

Reply:
No, this PR is simply to implement the field on the DB and ensure the field gets populated. Nothing is exposed in any view to either hiring staff or jobseekers. As outlined in the next steps, subsequent PRs will deal with implementing a new subjects field in the form and displaying this to users.